projects/Switch/packages/ishiiruka/package.mk

```diff
@@ -0,0 +1,54 @@
+################################################################################
```

Comment:
We can try to enable it for other platforms as requested on issue #511.

Reply:
I have a package ready for Dolphin and Ishiiruka, however the latter is not finished and I feel like we should wait before adding it. 

I wanted to submit a PR to update both cores at once, along with the new core info, but we can add Dolphin only in the meantime. Ishiiruka however is only there for perfs testing purposes.